lib/internal/cluster/child.js

```diff
@@ -278,10 +280,10 @@ Worker.prototype.destroy = function() {
 
   this.exitedAfterDisconnect = true;
   if (!this.isConnected()) {
-    process.exit(0);
+    process.exit(kNoFailure);
```

Comment:
the `k` prefixes are pretty weird, is it some kind of cpp convention?

Reply:
Yeah, that's the general enum naming convention in the code base (k = constant).

Reply:
Is that convention going to be exposed to users now in the form of these constants, though?

Reply:
For now the constants are only internal (there is no way to get access to `internalBinding('errors')` unless the user uses `--expose-internals`, which has no compatibility guarantee)